### PR TITLE
GAWB-1197 un-hardcode swagger version using sbt-buildinfo plugin magic

### PR DESCRIPTION
**Pull request by helgridly**
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Make sure Swagger is updated if API changes
- [x] **Submitter**: If updating admin endpoints, also update [firecloud-admin-cli](https://github.com/broadinstitute/firecloud-admin-cli)
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: JIRA ticket checks:
  - Acceptance criteria exists and is met
  - Note any changes to implementation from the description
  - Add notes on what you've tested
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Database checks:
  - If PR includes new or changed db queries, include the explain plans in the description
  - Make sure liquibase is updated if appropriate
  - If doing a migration, take a backup of the
    [dev](https://console.cloud.google.com/sql/instances/terraform-qfarbdq3lrexxck5htofjs5z6m/backups?project=broad-dsde-dev&organizationId=548622027621)
    and
    [alpha](https://console.cloud.google.com/sql/instances/terraform-r4caezzc35c4tb7pgdhwkmme4y/backups?project=broad-dsde-alpha&organizationId=548622027621)
    DBs in Google Cloud Console
- [x] **Submitter**: Update FISMA documentation if changes to:
  - Authentication
  - Authorization
  - Encryption
  - Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] Tell your tech lead (TL) that the PR exists if they want to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
- Review cycle:
  - LR reviews
  - Rest of team may comment on PR at will
  - **LR assigns to submitter** for feedback fixes
  - Submitter rebases to develop again if necessary
  - Submitter makes further commits. DO NOT SQUASH
  - Submitter updates documentation as needed
  - Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [x] **Assign to submitter** to finalize
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Squash commits and merge to develop
- [x] **Submitter**: Delete branch after merge
- [x] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [x] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [x] **Submitter**: Inform other teams of any API changes via hipchat and/or email
- [x] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -110,6 +113,11 @@ lazy val rawls = project.in(file("."))
     testOptions in Test ++= Seq(Tests.Filter(s => !isIntegrationTest(s))),
     testOptions in IntegrationTest := Seq(Tests.Filter(s => isIntegrationTest(s)))
   )
+  .settings(
+    buildInfoKeys := Seq[BuildInfoKey](name, version, scalaVersion, sbtVersion, "swaggerUIVersion" -> swaggerUIV),
```
Comment (jmthibault79) [inline build.sbt:117]: Do we need these other keys?  And why not name the val `swaggerUIVersion` to begin with?

Comment (helgridly) [reply build.sbt:117]: We don't need the other keys, but it seemed lonely on its own.

I was following convention of akkaV, sprayV etc.

**Comment by jmthibault79** [comment]
Tested in a local stack as-is and with updated Swagger version.  👍

File: build.sbt (hunk 1 above)
Comment (jmthibault79) [reply build.sbt:117]: I tried removing the string mapping, but it turns out to be necessary

Comment (helgridly) [reply build.sbt:117]: yeah, it needs to know what to call the val inside the case object it makes, and there's no way to do that without macros (or reflection, i suppose)

**Comment by davidangb** [comment]
Does this cause compile errors in IntelliJ?

Also, orchestration, agora, and thurloe all have the same pattern. If you're updating here, could you update those too?

**Pull request merged**